### ISSUE TITLE
User external kafka topics management 

### ISSUE THREAD

**Issue by nicksjl-adimian** [adimian/kirby#69]
Kirby user should be capable to manage the number of partitions/replications of his Kafka external topics. 
proposition of a solution: if needed, topics can be created by user outside Kirby. By default, if no topic is created by user, topic is created by Kirby with a number of partitions set to 5 and a replication factor set to 2 (arbitrary number).

**Comment by PierreSavatte** [comment]
The  creation of the `create_kafka_topic(name=topic_name, nb_partitions=5, replications=2)` function that you proposed in the issue #70 is a good idea in my opinion.

This function can create the topic if it don't already exist. What do you think?

**Comment by nicksjl-adimian** [comment]
Exactly. I suggest to use the function already existing in contest.py file and to adapt it.
